react/features/base/conference/middleware.js

```diff
@@ -152,6 +154,13 @@ StateListenerRegistry.register(
 function _conferenceFailed(store, next, action) {
     const result = next(action);
 
+    // XXX After next(action), it is clear whether the error is recoverable.
```

Comment:
can you move this comment to where the recoverable flag is checked? it feels out of place here

Reply:
Agree! It's fixed now!